api/wallet.go

```diff
@@ -190,7 +190,14 @@ func (api *API) walletInitHandler(w http.ResponseWriter, req *http.Request, _ ht
 	if req.FormValue("encryptionpassword") != "" {
 		encryptionKey = crypto.TwofishKey(crypto.HashObject(req.FormValue("encryptionpassword")))
 	}
-	seed, err := api.wallet.Encrypt(encryptionKey)
+
+	var seed modules.Seed
+	var err error
+	if req.FormValue("force") == "true" {
+		seed, err = api.wallet.Reencrypt(encryptionKey)
```

Comment:
The advantage of this design is that it can be used to add a `"force"` flag to the `InitFromSeed` endpoint as well.